transactor/src/main/scala/io/mediachain/copycat/TransactorService.scala

```diff
@@ -129,17 +129,24 @@ extends ClientStateListener with JournalListener {
   }
   
   private def journalCommitEvent(entry: JournalEntry) {
-    if (state.streaming) {
+    if (state.streaming && entry.index >= state.index) {
+      if (entry.index != state.index) {
+        logger.warn(s"Commit skipped entries ${state.index} -> ${entry.index}")
```

Comment:
The error message indicates that our index jumped, so there must be some skipped entries.
(again, this should not happen but famous last words)